src/components/10-atoms/text/ui.test.js

```diff
@@ -184,3 +184,62 @@ test('should update pure text dynamically and wrap in <p>', async t => {
   await t.expect(await $axaElem.innerText).contains(oldText);
   await t.expect(await $axaElem.innerHTML).contains('<p>');
 });
+
+fixture('Text - React')
+  .page(
+    `${host}/iframe.html?id=components-atoms-text-react-demos--feature-text-with-dynamic-children-under-react`
+  )
+  .beforeEach(async t => {
+    await t.resizeWindow(800, 600);
+  });
+
+test('should update dynamically for pure and HTML texts', async t => {
```

Comment:
Dont you want to split this huge test into 2? One for pure and the other for html texts.

Reply:
I interpret this request as "don't you want to make this test shorter, more readable" and DRYify it internally. Hope that satisfies the wish you had here.

Done.